transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/TransportClientConfig.java

```diff
@@ -86,6 +86,34 @@ public static boolean getConnectionKeepAlive() {
         .get();
   }
 
+  public static int getHttp1ConnectionKeepAliveTimeoutInSeconds() {
+    int result = DynamicPropertyFactory.getInstance()
+            .getIntProperty("servicecomb.rest.client.http1.connection.keepAliveTimeoutInSeconds", -1)
+            .get();
+    if (result >= 0) {
+      return result;
+    }
+    result = getConnectionIdleTimeoutInSeconds();
+    if (result > 1) {
+      return 0;
```

Comment:
why return 0?

Reply:
because HttpClientOptions will check value mast above 0 when setting value.
![image](https://user-images.githubusercontent.com/97039406/217131820-95017f79-dd5a-4951-bdb1-f1215ff919e9.png)

Reply:
fixed